apiserver/config.go

```diff
@@ -0,0 +1,41 @@
+package apiserver
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/glog"
+
+	"istio.io/manager/model"
+)
+
+// Config is the complete configuration including a parsed spec
+type Config struct {
```

Comment:
can you please remove `inputDoc` from `istioctl` and use this `Config` instead there?